camus-shopify/script/hive-generic-partitioner.py

```diff
@@ -23,6 +23,7 @@
                                         all directories found in <datadir> will be partitioned.
     -o --hive-options=<options>         Any valid Hive CLI options you want to pass to Hive commands.
                                         Example: '--auxpath /path/to/hive-serdes-1.0-SNAPSHOT.jar'
+    -i --ignore_before_date=<date>      Do not create Hive partitions before a certain start date, default is None
```

Comment:
@drdee what was the use case here? preventing creations all the way back to e.g. 1970?